src/Microsoft.AspNetCore.Mvc.Core/Authorization/AuthorizeFilter.cs

```diff
@@ -75,7 +75,6 @@ public AuthorizeFilter(AuthorizationPolicy policy)
 
             // Note: Default Anonymous User is new ClaimsPrincipal(new ClaimsIdentity())
             if (httpContext.User == null ||
```

Comment:
Still can't, at least not with our DefaultHttpContext, but who knows what implementation is under this HttpContext... :-)